guide/slash-commands/permissions.md

```diff
@@ -0,0 +1,74 @@
+# Slash command permissions
+
+Slash commands have their own permissions system. This system allows you to set the default level of permissions required for a user to to execute a command when its first deployed or your bot is added to a new server.
+
+The slash command permissions for guilds are defaults only and can be altered by guild administrators, allowing them to configure access however best suits their moderation and server roles. Your code should not try to enforce its own permission management, as this can result in a conflict between the server-configured permissions and your bot's code.
+
+::: warning
+It is not possible to prevent users with Administrator permissions from using any commands deployed to their guild. Think twice before creating "dev-only" commands such as an `eval`.
+:::
+
+## Member permissions
+
+You can use <DocsLink section="builders" path="class/SlashCommandBuilder?scrollTo=setDefaultMemberPermissions" type="method" /> to set the default permissions required for a member to run the command. Setting it to `0` will prohibit anyone in a guild from using the command unless a specific overwrite is configured or the user has the Administrator permission flag.
+
+For this, we'll look at two common moderation commands, `ban` and `kick`. For a ban command, a sensible default is to ensure that users already have the Discord permission `BanMembers` in order to execute this command.
+
+```js {9}
+const { SlashCommandBuilder, PermissionFlagsBits } = require('discord.js');
+
+const data = new SlashCommandBuilder()
+	.setName('ban')
+	.setDescription('Select a member and ban them.')
+	.addUserOption(option =>
+		option
+			.setName('target')
+			.setDescription('The member to ban')
+			.setRequired(true))
+	.setDefaultMemberPermissions(PermissionFlagsBits.BanMembers);
```

Comment:
odd indent + highlight (supposedly this should highlight the permissions, right?)

Reply:
Highlight is definitely wrong. The indenting is correct though from what I can see. It's in line with name, description, user option

Reply:
ah right, perms belong to one tier above, yep
address highlight then this is good to resolve